### ISSUE TITLE
Fractured phalanx of pes

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11184]
**HPO term**
https://hpo.jax.org/browse/term/HP:0041240

**Synonym to be added**
toe phalanx fracture

(pes is a bit arcane)

**broad/exact/narrow/plural/abbreviation ?**

**Comment by pnrobinson** [comment]
@MickeySegal 
Agree, I have updated the label!
